src/sentry/models/event.py

```diff
@@ -353,11 +354,53 @@ def prev_event(self):
             group_id=self.group_id,
         ).exclude(id=self.id).order_by('-datetime')[0:5]
 
-        events = [e for e in events if e.datetime == self.datetime and e.id < self.id
-                  or e.datetime < self.datetime]
+        events = [e for e in events if e.datetime == self.datetime and e.id < self.id or
+                  e.datetime < self.datetime]
         events.sort(key=EVENT_ORDERING_KEY, reverse=True)
         return events[0] if events else None
 
+    @classmethod
+    def get_event(cls, id_or_event_id, project_id=None):
```

Comment:
This maybe belongs better in a `Manager` class but it doesn't look like `Event` has its own custom manager, so I'm not sure if I should create one just for this.

Reply:
I would, because I find out mix-match use of manager-like methods without-managers very confusing. But I could be alone. I'd also lean toward something like `get_by_id` or something since `get_event` itself is wide open.

Reply:
Yeah, I see that `bind_nodes()` is also in the `BaseManager` whereas it really only applies to Events, so I should probably move that into an Event-specific Manager class too. 

Of course we already have an `EventManager` which is completely separate from this, for maximum confusion